src/com/facebook/buck/jvm/kotlin/KotlincToJarStepFactory.java

```diff
@@ -434,6 +444,21 @@ private String encodeOptions(Map<String, String> options) {
     }
   }
 
+  private String getFriendsPath(
+      SourcePathResolver sourcePathResolver, ImmutableList<SourcePath> friendPathsSourcePaths) {
+
+    // https://youtrack.jetbrains.com/issue/KT-29933
+    ImmutableSortedSet<String> absoluteFriendPaths =
+        ImmutableSortedSet.copyOf(
+            friendPathsSourcePaths
+                .stream()
+                .map(path -> sourcePathResolver.getAbsolutePath(path).toString())
+                .collect(Collectors.toSet()));
+
+    return "-Xfriend-paths="
+        + absoluteFriendPaths.stream().reduce("", (path1, path2) -> path1 + "," + path2);
```

Comment:
What about the same when the list is empty, it still adds `-Xfriend-paths=` option?

Reply:
Added a check for that, tks :)